### ISSUE TITLE
Training a model

### ISSUE THREAD

**Issue by PEE8** [fizyr/keras-retinanet#1234]
![img1](https://user-images.githubusercontent.com/14888010/71188313-bd0eca00-22a6-11ea-8b6e-2780ebe6138b.png)
![img2](https://user-images.githubusercontent.com/14888010/71188330-c0a25100-22a6-11ea-912d-476bafec3b0c.png)

I am training the model with train.py of (oct 2018 version). It trains and gives loss, regression_loss, classification_loss but stops after 2 epochs with the errors as shown in the attachments. Can you tell me how to resolve it please.

**Comment by game-sys** [comment]
try to run other than c drive

**Comment by PEE8** [comment]
Thanks I was able to train the model.
Now to convert the model - Is this syntax correct-
python keras_retinanet/bin/convert_model.py snapshots/name_of_model_created.h5 Models/output.h5 where output.h5 converted model will be used for prediction.

**Comment by PEE8** [comment]
I was able to get models by training and also was able to convert the model.
